plugin/net/cni.go

```diff
@@ -70,28 +77,31 @@ func (c *CNIPlugin) CmdAdd(args *skel.CmdArgs) error {
 	if err != nil {
 		return fmt.Errorf("unable to allocate IP address: %s", err)
 	}
+	// Only expecting one address
+	ip := result.IPs[0]
```

Comment:
Is "only expecting one address" guaranteed at this point?
Should we log a warning or return an error if we get more than one?